content/guides/contribute/index.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: Contribute
```

Comment:
I know this isn't a huge priority as the Contribute header in the nav isn't clickable, but is it possible to pull the content over from [/contribute](https://primer.style/contribute/) and put it here? 

We can leave out the link to [Primer CSS contribution guidelines](https://github.com/primer/css/blob/main/CONTRIBUTING.md)(as well as the link to Component Lifecycle since that will be it's own page).

Reply:
Thanks @emilybrick this is a great feedback! I was going to ask if we want to have some new content here or move over from the contribute repo but I forgot 🙈 Thanks for bringing this up. I pushed [a new commit](https://github.com/primer/design/pull/354/commits/e54e4560fa975d098b8a51a22b8e5757ff337d32) with the changes. Let me know if you have any concern.

>We can leave out the link to [Primer CSS contribution guidelines](https://github.com/primer/css/blob/main/CONTRIBUTING.md)(as well as the link to Component Lifecycle since that will be it's own page).

Not the case for React and ViewComponents? We still want to direct to the repo?